### PR TITLE
Upload new intro video.

### PR DESCRIPTION
**Pull request by waderyan**
Please take a look @gregvanl

### CODE REVIEW

File: docs/introvideos/quicktour.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+Order: 2
+Area: introvideos
+TOCTitle: Quick Tour using JavaScript
+ContentId: a51b707c-19f7-41af-b5c2-c296e6e9fea2
+PageTitle: Quick Tour of VS Code using JavaScript
```
Comment (gregvanl) [inline docs/introvideos/quicktour.md]: I think we want to use Visual Studio Code for the PageTitle and MetaDescription

File: docs/introvideos/quicktour.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+DateApproved: 10/03/2016
+MetaDescription: Quick Tour around VS Code using JavaScript.
+---
+
+# Quick Tour of VS Code using JavaScript
+
+What can you do with VS Code? This tutorial walks you through an overview of the core features of VS Code, including IntelliSense, debugging, Git version control integration, and more.
```
Comment (gregvanl) [inline docs/introvideos/quicktour.md]: For first mention in a topic use "Visual Studio Code".
Maybe "This tutorial gives you an overview ..."

File: docs/introvideos/quicktour.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+What can you do with VS Code? This tutorial walks you through an overview of the core features of VS Code, including IntelliSense, debugging, Git version control integration, and more. 
+
+<iframe src="https://www.youtube.com/embed/RAFFF1bIwTk?rel=0&amp;disablekb=0&amp;modestbranding=1&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+
+## Outline
+
+* See IntelliSense in action. 
+* Start the debugger and set a breakpoint. 
+* Commit code with Git integration. 
+* Find new features with extensions.
```
Comment (gregvanl) [inline docs/introvideos/quicktour.md]: Add new features with VS Code Extensions.

File: docs/introvideos/quicktour.md (hunk 1 above)
Comment (waderyan) [inline docs/introvideos/quicktour.md]: Done

File: docs/introvideos/quicktour.md (hunk 2 above)
Comment (waderyan) [inline docs/introvideos/quicktour.md]: Done.

File: docs/introvideos/quicktour.md (hunk 3 above)
Comment (waderyan) [inline docs/introvideos/quicktour.md]: Done.

**Comment by waderyan** [comment]
@gregvanl I pushed another commit. Please take a look and comment / merge :)

**Pull request merged**